docs/index.md

```diff
@@ -144,37 +144,49 @@ npm run kakunin
 In order to make cucumber steps autosuggestion work properly in JetBrains tools, make sure your project is `ECMAScript 6` compatible and you have `cucumberjs` plugin installed.
 Due to non-resolved issue in Jetbrains editors ([see here](https://youtrack.jetbrains.com/issue/WEB-11505)) we'll have to do one more step:
  
-Go to `step_definitions` directory 
+Go to `step_definitions` directory for each folder
 ```bash
-cd step_definitions
+cd web/step_definitions
+cd rest/step_definitions
+cd common/step_definitions
 ``` 
 
 Paste this code into terminal and restart your IDE:
 
 For Linux/MacOs:
 
 ```bash
-ln -s ../node_modules/kakunin/src/step_definitions/elements.ts kakunin-elements.ts
-ln -s ../node_modules/kakunin/src/step_definitions/debug.ts kakunin-debug.ts
-ln -s ../node_modules/kakunin/src/step_definitions/file.ts kakunin-file.ts
-ln -s ../node_modules/kakunin/src/step_definitions/form.ts kakunin-form.ts
-ln -s ../node_modules/kakunin/src/step_definitions/email.ts kakunin-email.ts
-ln -s ../node_modules/kakunin/src/step_definitions/generators.ts kakunin-generators.ts
-ln -s ../node_modules/kakunin/src/step_definitions/navigation.ts kakunin-navigation.ts 
-ln -s ../node_modules/kakunin/src/step_definitions/performance.ts kakunin-performance.ts 
+ln -s ../node_modules/kakunin/dist/web/step_definitions/elements.js kakunin-elements.js
```

Comment:
You can move the steps to folders (web, rest, common) inside the initialised project

/step_definitions
   /rest/step.js
   /web/step.js
   /common/step.js

Reply:
done